formats/link.js

```diff
@@ -4,6 +4,7 @@ class Link extends Inline {
   static create(value) {
     const node = super.create(value);
     node.setAttribute('href', this.sanitize(value));
+    node.setAttribute('rel', 'noopener noreferrer');
```

Comment:
Is noreferrer required? If I'm correct this was a workaround for FF, which was fixed in FF 52. This will cause issues with analytics reporting where traffic came from

Reply:
Hmm :thinking: I think it would be best as a config option to set that

Reply:
Would using a property such as `Link.REL = "noopener";` be efficient?

Reply:
According to [caniuse](https://caniuse.com/#search=noopener), the `rel=noopener` attribute is not supported by IE11, with `rel=noreferrer` at least being supported in Win 10 creators update versions of IE11. That's why I picked both attributes here.

> This will cause issues with analytics reporting where traffic came from
Personally I would consider this a feature. But I could understand if you didn't want to make this decision as part of your library.

Reply:
Ah my apologies, I should have checked IE support! This is good to go then. Thanks 😃

Reply:
Awesome :)
Please note though, that for a release that fixes the complete vulnerability, this commit as well as #2439 have to be included.